.vscode/settings.json

```diff
@@ -0,0 +1,10 @@
+{
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  "editor.formatOnType": false, // required
+  "editor.formatOnPaste": true, // optional
+  "editor.formatOnSave": true, // optional
+  "editor.formatOnSaveMode": "file", // required to format on save
+  "editor.inlineSuggest.suppressSuggestions": false,
+  "files.insertFinalNewline": true,
+  "files.trimTrailingWhitespace": true // optional but recommended
```

Comment:
Kommentarer er ikke gyldig i JSON, bør fjernes om denne filen skal sjekkes inn i versjoneringen:)

Og er det ok for alle at vi har felles vscode-settings for prosjektet? Bør være noe alle på teamet er med på.